.gitignore

```diff
@@ -253,3 +253,7 @@ tags
 # make temps
 bin/
 testbin/
+
+# bundle files+directories are generated
+bundle.Dockerfile
+bundle/
```

Comment:
I'm trying to understand the logic a bit here. Is the expectation that we don't want to check in the generated artifacts?

If so, is it safe to assume that we could regenerate these files at any time by just checking out the appropriate commit (or tag) and running the appropriate make target?

Reply:
Yes, the intention is that make command generates all the correct files from scratch.